src/templates/snek/gql/tasks/index.ts

```diff
@@ -0,0 +1,82 @@
+//#region > Imports
+//#PACKAGE "graphql"
+//## npm install "graphql"@14.6.0
+// Contains the interface for gql queries, mutations and subscriptions
+import { DocumentNode } from "graphql";
+
+//> Sessions
+// Contains the SNEK session
+import { SnekSession } from "../../../../session/sessions";
+//> Tasks
```

Comment:
I'd suggest to insert a new line above headers to improve readability.

Reply:
The current state of our style guide has no newline between headers. So this code part is currently correct, but might be changed with an style guide update.